cmsranking/static/TimeView.js

```diff
@@ -25,9 +26,14 @@ function format_time(time, full) {
     return (h + ":" + m + ":" + s);
 };
 
-function _get_time() {
-    // Return the seconds since January 1, 1970 00:00:00 UTC
-    return $.now() / 1000;
+function _get_server_time() {
+    // Return the seconds since January 1, 1970 00:00:00 UTC (server time)
+    return ($.now() / 1000) - TimeView.diff_with_server_time;
+}
+
+function _get_server_timezoned_time(server_utc_offset) {
+    var local_utc_offset = new Date().getTimezoneOffset() * -60;
+    return _get_server_time() + server_utc_offset - local_utc_offset;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'server_utc_offset' is not in camel case.](https://www.codacy.com/app/s-maggiolo/cms/file/2192640431/issues/source?bid=3167167&fileBranchId=3167181#l36)